docs/source/algorithms/AlignComponents-v1.rst

```diff
@@ -55,110 +75,6 @@ example below).
 The source and sample positions (in that order) are aligned before any
 components are aligned.
 
-Usage
```

Comment:
Can we add the usage example back (and update it)?  It would be helpful to get users started if they are not familiar with this process.

Reply:
`AlignComponents` is not meant to be used directly by the users. Only `CorrelliPowderCalibrationCreate` uses this algorithm because only `CorrelliPowderCalibrationCreate` is able to read the output from `PDCalibration` (a table of peak positions in d-spacing units) and create the input that  `AlignComponents` needs (a table of peak positions in TOF units).

In the past, `AlignComponents` required a table of DIFC values, produced by `PDCalibration` so that a user could pass this table as input. Now it's kind of a private algorithm of `CorelliPowderCalibrationCreate`, hence the lack of examples.

Reply:
I see. In this case, we will use the unittest as the example for developers.